db/migrate/20230710044802_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```

Comment:
- [ ] [OPTIONAL]
It will be a good idea to make the bio's data type _text_ instead of _string_ to hold larger lines of text.